airgun/entities/contenthost.py

```diff
@@ -51,14 +55,20 @@ def execute_package_action(self, entity_name, action_type, value):
         :param value: Package or package group group name to remotely
             install/upgrade/remove (depending on `action_type`)
 
+        :param installed_via: what installation method was used
```

Comment:
Consider mentioning what the expected values are for this field.

Reply:
Consider also setting the default to `rex` instead of `None`, to be more explicit in the param and in the check against the param.